### ISSUE TITLE
False detection?

### ISSUE THREAD

**Issue by peerjuettner** [rowanwins/point-in-polygon-hao#13]
Hi
I'm trying to test if the following point is inside my polygon:
`
const myPoly = [[[1234,2292],[1834,2292],[1834,1692],[1234,1692]]]

test('is inside myPoly', t => {
    t.false(inside([430, 2205], myPoly))
});
`
The test always fails as the library returns

 `true`
I tried visualizing the points and I think the test should return false.
![grafik](https://user-images.githubusercontent.com/18301629/112451325-a979cb00-8d55-11eb-8b42-af5491ae58bc.png)
Is there an error in my syntax or is the libary returning a wrong result?

**Comment by rowanwins** [comment]
Hi @peerjuettner 

You need to make sure the first and last points in your polygon are the same. 
````

const myPoly = [[[1234,2292],[1834,2292],[1834,1692],[1234,1692]]]
// becomes
const myPoly = [[[1234,2292],[1834,2292],[1834,1692],[1234,1692], [1234,2292]]]
````

I think I will write some handling in the library for this to either auto-fix or at least throw an error for it.

**Comment by rowanwins** [comment]
The input validation is now included in release v1.0.0
